android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIDialog.java

```diff
@@ -289,6 +289,7 @@ public void onCancel(DialogInterface dlg)
 			dialog = getBuilder().create();
 			dialog.setCanceledOnTouchOutside(
 				proxy.getProperties().optBoolean(TiC.PROPERTY_CANCELED_ON_TOUCH_OUTSIDE, true));
+			dialog.setCancelable(proxy.getProperties().optBoolean(TiC.PROPERTY_BUTTON_PRESS_REQUIRED, true));
```

Comment:
The default value should be `false` to retain the old behavior.

You should also `!` the property's returned value since `buttonPressRequired` set to `true` means it is not cancelable via back key or outside taps.